.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "solidity-sdk/lib/openzeppelin-contracts"]
 	path = solidity-sdk/lib/openzeppelin-contracts
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+[submodule "protocol/lib/forge-std"]
+	path = protocol/lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
+[submodule "protocol/lib/openzeppelin-contracts"]
+	path = protocol/lib/openzeppelin-contracts
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts
```

Comment:
@abresas recently cleaned up excessive submodules - all foundry projects now depend on the same submodules using symlinks. Could you do the same here?